### ISSUE TITLE
Jetbrains icons look out of place

### ISSUE THREAD

**Issue by heyhippari** [numixproject/numix-core#3706]
The redisigned Jetbrains icons have a wild gradient that goes way out of the 10L guideline.

AFAIK, they're the only icons to do that and, as a result, look quite out of place.

I suggest averaging the two colors of the current gradient and then applying a 10L gradient as usual.

Examples:
![idea-old](https://cloud.githubusercontent.com/assets/19396809/25564418/3eb26a1c-2db3-11e7-9a33-25952f295868.png) => ![idea-new](https://cloud.githubusercontent.com/assets/19396809/25564420/44045912-2db3-11e7-9ada-c3dc0ee136c2.png)

**Comment by palob** [comment]
When we merged those we intentionally opted for a 2-colour-gradient as there was no single characteristic colur in the oiginal icons. There are a few more icons with a 2-colour gradient, for instance curlew and soundnode.

**Comment by Foggalong** [comment]
Ideally I would like to get rid of dual gradients in all icons, so I'm gonna leave this open. If anyone does have any good ideas how to make this happen so that it works across all the dual gradient icons I'd love to hear them.

@MrTimscampi I'd be interested to know if the averaging works for all the Jetbrains ones

**Comment by heyhippari** [comment]
@Foggalong I haven't tested all of them, but PyCharm and IntelliJ work pretty well. Stuff that has wildly different colors might be less perfect, though.

I'm busy until early July with finals and internship/thesis presentations, but I'll try to make a few of them if/when I have time, if you want.

**Comment by heyhippari** [comment]
Update on this, here are some tests for the most jarring ones I found.

**Rubymine**:
![rubymine old](https://user-images.githubusercontent.com/19396809/29824510-0ba28fb6-8cd2-11e7-867f-399237b2977c.png) -> ![rubymine](https://user-images.githubusercontent.com/19396809/29824189-14618cfc-8cd1-11e7-8fa2-c0137b26e671.png)

**Rider**:
![rider old](https://user-images.githubusercontent.com/19396809/29824522-13762b80-8cd2-11e7-9ec6-623c4389ecfd.png) -> ![rider](https://user-images.githubusercontent.com/19396809/29824529-172bed28-8cd2-11e7-9ca7-bd13d4c78f81.png)

If the 10L guideline is still applicable (I saw that the guidelines changes a bit while I was away :p ) and those look alright, I can make a PR with everything.

**Comment by Foggalong** [comment]
@MrTimscampi There haven't been any guideline changes that I'm aware of? But anyways submit a PR and we'll see what we can do about the jarring ones :)

**Comment by Foggalong** [comment]
@MrTimscampi Are you still submitting a PR with these changes?

**Comment by jgandert** [comment]
I've created svg icons for PyCharm and IntelliJ IDEA: [idea and pycharm icons.zip](https://github.com/numixproject/numix-core/files/1594345/idea.and.pycharm.icons.zip)

This is what they look like in Unity:

![screenshot from 2017-12-30 13-04-24](https://user-images.githubusercontent.com/3063858/34454031-312fce60-ed62-11e7-84fb-4e635d7fffa7.png)


Font used is [Ubuntu Mono](https://design.ubuntu.com/font/).

and these are the original icons for comparision: 
![original idea icon for comparision](https://user-images.githubusercontent.com/3063858/34453962-5d8a447e-ed60-11e7-9d73-2534d5bac9f7.png) ![original pycharm icon for comparision](https://user-images.githubusercontent.com/3063858/34453963-5da7ab7c-ed60-11e7-8778-ccfb22dd3672.png)

**Comment by palob** [comment]
They look cool but this would make them stand out even more.

**Comment by jgandert** [comment]
@palob True. I used a Numix Square icon as a base.. Maybe it would be possible to modify them to fit in better. Whoever wants to modify them to fit in and push the work to one of Numix repositories, feel free.

**Comment by bkanuka** [comment]
As @palob mentioned in #4157 , this is blocking adoption of Jetbrains new rebranding (to all 2-letter icons).

The multicolor background is part of Jetbrains branding and is recognizable. I think for that reason the multicolor background should stay. @palob basically mentioned this early in the thread and others seemed to agree.

**Comment by Foggalong** [comment]
@bkanuka If the concern is that it contradicts branding, then it's a moot point. The whole idea of universal icon themes are to override the design decisions of each app developer and standardise to a consistent design theme. Whether or not you agree with that is a topic of debate (Sam Hewitt of Moka & Suru fame did a [great blog](https://samuelhewitt.com/blog/2017-11-26-linux-themes-third-party-icons) on the topic. Personally disagree with his points, but it's worth a read nonetheless) but if you're using a universal icon theme then there's the implicit assumption that that's what it'll do.

**Comment by bkanuka** [comment]
@Foggalong you make a good point. Really though I'm less concerned with numix contradicting branding, than I am with it contradicting *itself*. Currently jetbrains icons are different between square and circle variants (the circle icons all use 2-letter style). Among the square icons, some jetbrains icons use 2-letter style, and others use an older icon style.

A decision needs to be made here before jetbrains icons can be made consistent throughout (see #4157). I have an opinion, as stated above, but that's secondary ;-)

**Comment by Foggalong** [comment]
@bkanuka Oh don't worry about that - there's a series of fixes currently in the works to standardise the styles across Circle & Square. The distinctions now are just a holdout from the theme having not been updated for a while :smile:

**Comment by palob** [comment]
Did those apps' icons stop to be hardcoded? From #4198 it seems the icon names are simply ``jetbrains-xxx`` now.

**Comment by palob** [comment]
Are we ready for a decision?

@janoj You are the one who did the redesign. Do you remember which font you used or did you come up with the paths from scratch?

**Comment by Foggalong** [comment]
Yeah [what was said here](https://github.com/numixproject/numix-core/issues/3706#issuecomment-302231775) still stands. I'd like to move to move away from two tone gradients where possible :)

**Comment by Foggalong** [comment]
@MrTimscampi Would you still be up for submitting a PR to solve this or do you want us to handle it?

**Comment by heyhippari** [comment]
@Foggalong I don't really use Numix or Jetbrains software anymore, so feel free to handle it :)

**Comment by Foggalong** [comment]
@MrTimscampi Sure thing :+1:

**Comment by palob** [comment]
One more thing to do: 

+ The black squares should be shrinked to ``24x24`` (if possible for all, size  should stay uniform) and be brightened up to ``L=26``.
